### PR TITLE
avm2: Convert Capabilities, System to AS

### PR DESCRIPTION
**Pull request by adrian17**
Exactly as stub-y as before, now in AS :)
No functional changes intended.

### CODE REVIEW

**Review by Herschel — APPROVED** [review]
Thank you!

**Review by relrelb — APPROVED** [review]

File: core/src/avm2/globals/flash/system/System.as [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package flash.system {
+    public final class System {
+        public static function gc(): String {
```
Comment (adrian17) [inline core/src/avm2/globals/flash/system/System.as:3]: Oops don't merge it please, wrong return type :)

Comment (torokati44) [reply core/src/avm2/globals/flash/system/System.as:3]: Quiet ping for this one-word fix. :no_mouth:

Comment (torokati44) [reply core/src/avm2/globals/flash/system/System.as:3]: never mind, sorry 😔

Comment (adrian17) [reply core/src/avm2/globals/flash/system/System.as]: fix'd

**Pull request merged**